workshops/slack-pfp/README.md

```diff
@@ -0,0 +1,309 @@
+
+  
+
+On this screen, scroll down to the `User Token Scopes` section and select the button that says `Add an OAuth Scope`.
+
+  
+
+![OAuth Scope Selection on Slack API page](https://cloud-1xkvojjl4.vercel.app/0screenshot_2020-11-19_at_11.55.09_pm.png)
+
+  
+
+In the pop-up box that follows, select `users.profile:write` and add that scope.
+
+  
+
+![OAuth Scope Selection on Slack API page completed](https://cloud-2ru7t5yz0.vercel.app/0screenshot_2020-11-20_at_12.00.03_am.png)
+
+  
+
+Scroll to the top of the page, click `Install to Workspace`, then `Allow` and then you will receive a token in return.
+
+  
+
+![Sample Token from Slack](https://cloud-1spwslhyd.vercel.app/0screenshot_2020-11-20_at_12.05.54_am.png)
+
+  
+
+Copy the generated token and store it in a safe space. Don't share it with anyone else (this is a fake token in the image, don't worry).
+
+  
+
+## Building our program
+
+To get started you are going to want to create a new Node.js  [repl.it](http://repl.it/)  project, [click here to do so]([https://repl.it/languages/nodejs](https://repl.it/languages/nodejs)).
+
+Let's get going by adding your Slack token to your environment variables. What are environment variables? These are super secret variables that you don't want to store publicly. 
+
+Create a `.env` file and fill it in like:
+
+```
+SLACK_TOKEN=XOXP-YOUR-TOKEN
+```
+
+Remember we want to keep this a secret, so create a `.gitignore` file and make it's contents:
+
+```
+.env
+```
+
+Now, let's add two packages (`axios` & `@slack/web-api`) to our `index.js` file.
+
+```javascript
+const { WebClient } = require('@slack/web-api');
+const axios = require('axios').default;
+```
+
+Axios will be used to fetch the images from URLs and the Slack Web API package will be used to interact with Slack.
+
+Next, let's create a data object with each of our images. Replace each URL with your own to make this program your own.
+
+```javascript
+const images = {
+	"morning": "https://images.unsplash.com/photo-1517188206596-1e1f7c954177?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1650&q=80",
+	"afternoon": "https://images.unsplash.com/photo-1566452348683-91f934cd9051?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=2090&q=80",
+	"night": "https://images.unsplash.com/photo-1519446251021-1c4ee77fec1e?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=913&q=80"
+}
+```
+
+### Setting our profile picture
+
+Now that we've got our images we need to create an async function that will handle setting our profile picture.
+
+```javascript
+async function setPFP() {  
+	console.log(images)
+}
+```
+
+An async function is important as it allows us to wait for a line of code to complete before moving to the next line using the `await` keyword.
+
+What should we do first? How about setting a profile picture! Inside our `setPFP` function add the following lines. This will fetch the contents of the image r
```

Comment:
```suggestion
What should we do first? How about setting a profile picture! Inside our `setPFP()` function add the following lines. This will fetch the contents of the image.
```